packages/react/src/components/MenuBar/index.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import axe from '../../axe';
+import MenuBar from '../MenuBar';
+import { TopBarItem } from '../TopBar';
+
+const [right, left] = [39, 37];
+const child = (
+  <TopBarItem>
+    <div />
+  </TopBarItem>
+);
+
+test('renders', () => {
+  render(<MenuBar>{child}</MenuBar>);
+  expect(screen.getByRole('menubar')).toBeTruthy();
+});
+
+test('supports falsy children', () => {
+  const { container } = render(
+    <MenuBar>
+      {child}
+      {false && child}
+    </MenuBar>
+  );
+  expect(container).toBeTruthy();
+  expect(container.querySelectorAll('div').length).toBe(1);
+});
+
+test('handles left arrow', async () => {
+  const { getAllByRole } = render(
```

Comment:
I would recommend using `screen.getByRole` here because that does 2 things, it validates that we're rendering the element in the document, there's only 1 type of that element, and that we should be firing events on the menubar itself.

Reply:
added thin\wide state tests
using scree.getAllByRole to get needed elements